app/services/reminder_queuer.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+#
+# Class to enqueue reminder email jobs
+class ReminderQueuer
+  # Create new instance of ReminderQueuer, call #queue on it
+  def self.queue(**params)
+    new(**params).queue
+  end
+
+  # Initialize a new ReminderQueuer
+  #
+  # @param draw [Draw] the draw to send reminder emails in
+  # @param type [String] type of email to send; must be "intent" or "locking"
+  def initialize(draw:, type:)
+    @draw = draw
+    @type = type
+  end
+
+  # Queue the email job
+  def queue
+    ActiveRecord::Base.transaction do
+      draw.update!(email_type: type, last_email_sent: Time.zone.now)
+      queue_job
```

Comment:
Does this hit the database via ActiveRecord? If not, we can put this (and `success`) outside of the transaction.